src/pages/events/createEvents/CriteriasPaper.tsx

```diff
@@ -48,190 +304,627 @@ const CriteriasPapers = () => {
-              <Box className={classes.criteriasPaperHeaderBox}>
-                <Box className={classes.checkboxManagerBox}>
-                  <Typography className={classes.criteriasPaperHeaderText}>
-                    Rating Score Range
-                  </Typography>
                 </Box>
-                <Button className={classes.criteriaAddButton} variant="text" disableRipple>
-                  <Typography className={classes.criteriasAddButtonText}>Add</Typography>
-                </Button>
-              </Box>
-              <Box className={classes.criteriasPaperInfo}>
-                <FormControl className={classes.scoreRangeBox}>
-                  <RadioGroup
-                    aria-labelledby="demo-radio-buttons-group-label"
-                    defaultValue="female"
-                    name="radio-buttons-group"
-                  >
-                    <FormControlLabel
-                      className={classes.ratingScoreLabel}
-                      value={5}
-                      control={<BpRadio />}
-                      label="From 1 point - to 5 points"
-                    />
-                    <FormControlLabel
-                      className={classes.ratingScoreLabel}
-                      value={10}
-                      control={<BpRadio />}
-                      label="From 1 point - to 10 points"
+              ))}
+              {newSubFilters.map((newSubFilter: ISubCriteria, indexSubFilter: number) => (
+                <Box key={indexSubFilter} className={classes.eachSubCriteria}>
+                  <Box className={classes.eachCriteria} key={indexSubFilter}>
+                    <Checkbox
+                      className={classes.checkbox}
+                      defaultChecked
+                      icon={<UnCheckedIcon />}
+                      checkedIcon={<CheckedIcon />}
                     />
-                    <FormControlLabel
-                      className={classes.ratingScoreLabel}
-                      value={20}
-                      control={<BpRadio />}
-                      label="From 1 point - to 20 points"
+                    <TextField
+                      onChange={(e) => {
+                        const tempObj = {...newSubFilter, name: e.target.value}
+                        const newArr = newSubFilters.map((item: ISubCriteria) => {
+                          if (item.id === tempObj.id) {
+                            return tempObj
+                          } else {
+                            return item
+                          }
+                        })
+                        setNewSubFilters(newArr)
+                      }}
+                      InputProps={{
+                        classes: {notchedOutline: classes.noBorder},
+                      }}
+                      value={newSubFilter.name}
+                      className={classes.outlinedBasic}
+                      placeholder="Subfilter name"
```

Comment:
Component is too big, can we move parts to other components?

Reply:
done